pkg/aws/cloudwatch.go

```diff
@@ -196,6 +200,8 @@ func (c CloudWatchClient) GetOneDayStatistics(tg string, startTime, endTime time
 		return nil, 0, nil
 	}
 
+	fmt.Println(result.MetricDataResults)
```

Comment:
May be, using logger is better than use fmt.Print :)

Reply:
Oh..! wrong habit.. 
I fixed it :)

Reply:
Thank you. :)